hibernate-testing/src/main/java/org/hibernate/testing/orm/junit/Jpa.java

```diff
@@ -108,6 +109,8 @@
 	 */
 	boolean loadByIdComplianceEnabled() default false;
 
+	Class<? extends StatementInspector> statementInspectorClass() default StatementInspector.class;
```

Comment:
Ihad already added a `SettingProvider[] settingProviders() default {};` to `@Jpa` in [https://github.com/hibernate/hibernate-orm/pull/4322/](https://github.com/hibernate/hibernate-orm/pull/4322/), since that also covers the need to pass a StatementInspector class, I may simply revert this in my PR, wdyt?